src/components/search/SearchResult.vue

```diff
@@ -2,36 +2,33 @@
     <div>
         <div class="row p-2 align-items-center">
                 <div class="text-muted text-sm mr-auto">Query time: {{ searchData.queryTime }} ms</div>
-                <div class="text-primary text-sm">Results found: {{ searchData.numFound }}</div>
                 <div class="col-sm-2">
                     <b-input-group size="sm">
                         <b-form-select v-model="perPage" :options="options"/>
                     </b-input-group>
                 </div>
-                <!-- <div class="col-sm-3"> -->
-                    <b-pagination class="mt-3" size="sm" align="center" :total-rows="searchData.numFound" v-model="currentPage" :per-page="perPage">
-                    </b-pagination>
-                <!-- </div> -->
+                <b-pagination class="mt-3" size="sm" align="center" :total-rows="searchData.numFound" v-model="currentPage" :per-page="perPage">
+                </b-pagination>
         </div>
-        <div class="alert elevation-0 mb-2 p-1 text-center text-sm" role="alert"
+        <div class="alert elevation-0 mb-2 p-1 text-left text-sm" role="alert"
              :class="getAlertClass()">
                 {{getRestatedText()}}
         </div>
-        <div class="bg-gray elevation-0 mb-2 p-1">
-            <div class="row pb-1">
+        <div class="elevation-0 mb-2 p-1">
+            <div class="row">
                 <div class="col-6">
-                    <span class="text-bold text-sm">Gene family</span>
+                    <span class="text-bold">Gene family</span>
```

Comment:
ask peifen about blue border bottom and top

Reply:
peifen is ok with no blue border. But wants the font to be bold and larger. And the banner to have larger height.